style.css

```diff
@@ -1,7 +1,396 @@
 .basis {
+  background-color: #e3eafe;
   font-family: "Roboto", serif;
   font-weight: 900;
-  height: 2766px;
-  width: 1440px;
+  margin: 0;
+  max-width: 1440px;
+  max-height: 2766px;
+  min-width: 320px;
 }
-\n
\ No newline at end of file
+
+.header {
+  background-image: url(images/nav/pic-bgr.jpg);
+  max-height: 839px;
```

Comment:
Codacy found an issue: [Expected newline before "}" of a multi-line block (block-closing-brace-newline-before)](https://app.codacy.com/gh/WebHeroSchool/IFW.2/pullRequest?prid=6289680)